CHANGELOG.md

```diff
@@ -204,5 +212,5 @@
 
 ## 2020-10-09
 ### Added
-- Add `SHARELATEX_PORT` option to `overleaf.rc` file, which defaults
+- Add `OVERLEAF_PORT` option to `overleaf.rc` file, which defaults
```

Comment:
This looks like a historic entry that we might want to keep as is?

Reply:
That slipped, thanks!